crypto/asymmetric/keypair.go

```diff
@@ -0,0 +1,16 @@
+package asymmetric
+
+import (
+	"github.com/btcsuite/btcd/btcec"
+	log "github.com/sirupsen/logrus"
+)
+
+func GenSecp256k1Keypair() (privateKey *btcec.PrivateKey, publicKey *btcec.PublicKey, err error) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function GenSecp256k1Keypair should have comment or be unexported